.github/workflows/ci.yml

```diff
@@ -0,0 +1,161 @@
+        chmod a+x apache-tomcat-$TOMCAT_VERSION/bin/*.sh
+        python ./checker.py build
+        python ./checker.py jar
+        java -jar ./build/dist/vnu.jar ./build/dist/index.html
+        java -jar ./build/dist/vnu.jar ./site/nu-about.html
+        jar xf ./build/dist/vnu.jar && file nu/validator/client/SimpleCommandLineValidator.class | grep "version 52.0"
+        python ./checker.py runtime-image
+        python ./checker.py war
+        jar xf ./build/dist-war/vnu.war && file WEB-INF/classes/nu/validator/servlet/VerifierServlet.class | grep "version 52.0"
+        apache-tomcat-$TOMCAT_VERSION/bin/catalina.sh start
+        cp ./build/dist-war/vnu.war apache-tomcat-$TOMCAT_VERSION/webapps/
+        sleep 15s; tail apache-tomcat-$TOMCAT_VERSION/logs/catalina.out || true
+        curl -sS 'http://localhost:8080/vnu/?out=gnu&doc=data:text/html;charset=utf-8,%3C%21doctype%20html%3E%3Cmeta%20charset%3Dutf-8%3E%3Ctitle%3ETest%3C%2Ftitle%3E' > RESULTS
+        test -z "$(if [ -t RESULTS ]; then cat RESULTS | grep -v '^$'; fi)"
+        if [ $? != 0 ]; then cat RESULTS; fi
+        python ./checker.py check
+        python ./checker.py test
+    - uses: actions/upload-artifact@v1
+      if: matrix.java == 11
+      with:
+        name: vnu.jar
+        path: "build/dist/vnu.jar"
+    - uses: actions/upload-artifact@v1
+      if: matrix.java == 11
+      with:
+        name: vnu.war
+        path: "build/dist-war/vnu.war"
+    - uses: actions/upload-artifact@v1
+      if: matrix.java == 11
+      with:
+        # Currently creates a nested zip https://github.com/actions/upload-artifact/issues/3
+        name: vnu.linux.zip
+        path: "build/dist/vnu.linux.zip"
+
+  windows:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+        - 8
+        - 11
+        - 13
+        - 14-ea
+        - 15-ea
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Java JDK
+      uses: actions/setup-java@v1.3.0
+      with:
+        java-version: ${{ matrix.java }}
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Update Submodules
+      run: python .\checker.py update
+    - name: Install Dependencies
+      run: python .\checker.py dldeps
+    - name: Build
+      run: |
+        python .\checker.py build
+        python .\checker.py jar
+        java -jar .\build\dist\vnu.jar .\build\dist\index.html
+        java -jar .\build\dist\vnu.jar .\site\nu-about.html
+        python .\checker.py war
+        python .\checker.py check
+        python .\checker.py test
+    - uses: actions/upload-artifact@v1
+      if: matrix.java == 11
+      with:
+        # Currently creates a nested zip https://github.com/actions/upload-artifact/issues/3
+        name: vnu.windows.zip
+        path: "build/dist/vnu.windows.zip"
+
+  darwin:
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+        - 11
```

Comment:
```suggestion
```

For our purposes, for OSX, we don’t need to specify any particular Java version. Whatever Java version it defaults to on OSX is fine.

Reply:
I'll check to see if there is one, but I'm not sure if OSX has a default JDK, or just a JRE version